versioned_docs/version-5.2.x/installation/npm.md

```diff
@@ -0,0 +1,78 @@
+---
+id: npm
+title: Install via NPM
+sidebar_label: Install via NPM
+---
+
+To get you started, we will show you the various options for installing the OpenHIM core along with the OpenHIM admin console.
+
+If you are installing the OpenHIM on ubuntu, then the installation process is very easy as we provide a debian package in the OpenHIE Personal Package Archive (PPA). Currently, the packages are only built for Ubuntu 14.04 but we hope to support the latest LTS soon. The OpenHIM in general supports all versions of Ubuntu.
+
+> **Note**: If you are installing the OpenHIM on a VM, please see [Installation using a Virtual Machine](#installation-using-a-virtual-machine) before proceeding with the installation of the OpenHIM.
+
+---
+
+## Manual Installation
+
+If you don’t have ubuntu or prefer to proceed with the installation manually, please follow the following steps.
+
+### Installing the OpenHIM Core
+
+The latest active LTS is **recommended**.
+
+> **Note**: libappstream3 may cause problems with the npm package manager if your ubuntu instance is not fully updated.
+
+1. Install the latest stable Node.js v8 or greater `curl -sL https://deb.nodesource.com/setup_6.x| sudo -E bash` then `sudo apt-get install -y nodejs`
+1. Install and start MongoDB 3.6 or greater. (If you are running Ubuntu 16.04, you may want to configure MongoDB as a systemd service that will automatically start on boot)
```

Comment:
@BMartinos Are there any changes for 18.04? I'm not familiar with systemd

Reply:
@MattyJ007 there any any specific changes to systemd in ubuntu 18 as far as im aware. should still work the same